modules/virt-additional-components.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-architecture.adoc
+
+:_content-type: CONCEPT
+[id="virt-additional-components_{context}"]
+= Additional {VirtProductName} components
+
+Intro text here.
+
+[cols="1,1"]
+|===
+|*Component* |*Description*
+
+|`deployment/hco-webhook`
+|Validates the `HyperConverged` custom resource contents.
+
+|`tekton`
+|Deploys additional tekton tasks that allow users to create VMs from templates, copy and modify templates, and create data volumes.
+
+|`deployment/tekton-tasks-operator`
+|Deploys Kubevirt tekton tasks and example pipelines that can work with VMs.
+
+|`daemonset/hostpath-provisioner`
+|Implements the legacy driver interface of the HPP.
+
+|`deployment/hyperconverged-cluster-cli-download`
+|Provides the virtctl tool binaries to the cluster so that you can download them directly from the cluster.
+
+|'pod/virt-launcher'
+|Contains the VM that was created by the user with `libvirt`.
+
+|'pod/cdi-importer'
+|Helper pod that exists long enough to perform a task, then completes and terminates.
```

Comment:
"pod that exists long enough to perform a task, then completes and terminates" is generically true for all pods everywhere.  @alitke can we describe cdi-importer pod better?

```suggestion
|Helper pod that is started when CDI is populating a PVC.
```